native/more/more-screen.react.js

```diff
@@ -213,6 +215,26 @@ class MoreScreen extends React.PureComponent<Props> {
               />
             </View>
           </View>
+          <View style={this.props.styles.slightlyPaddedSection}>
```

Comment:
I think there may have been a misunderstanding here. Did you uncomment this because you read my longer message and saw I mentioned that I don't like committing commented out code?

It's true I don't like committing "work-in-progress" code to master, and that includes commented out code. But since these buttons leads to work-in-progress functionality, they're definitely something that needs to be hidden in master. The reason is that when I make a new release, this stuff can't be in it. **My preference is simply to delete this sort of code and send it in a future commit, but in this specific case I'm okay with it being commented out.**

Separately, I'd appreciate any advice you have on how I can communicate better. In my last comment, I tried to **emphasize that we didn't need to do any more changes** in anything other than `RelationshipList`, because I was scared we'd up needing more **review cycles**:

> Don't worry about making any changes based on what I say below... I just wanted to make sure I communicate well.

> Let's definitely not worry about changing it now.

Maybe I should've bolded these parts. Sometimes I worry that I'm writing too much, and the important stuff gets lost.